server/serverAPI/controller/BaseController.ts

```diff
@@ -0,0 +1,27 @@
+import { Response } from "express";
+import ResponseFormatter from "../../utils/ResponseFormatter";
+import { ResponseTypes } from "../../utils/ResponseTypes";
+
+export default class BaseController {
+    protected getException(error: unknown): string {
+        if (error instanceof Error) {
+            return error.message;
+        }
+
+        return String(error);
+    }
+
+    protected sendError(statusCode: number, res: Response, message?: any): Response<any, Record<string, any>> {
```

Comment:
This line (and some others) are too long. Please try to keep to < 100 chars per line.